src/is_subset.jl

```diff
@@ -499,3 +503,40 @@ Check whether an interval is contained in another interval.
 function ⊆(x::Interval, y::Interval)
     return x.dat ⊆ y.dat
 end
+
+
+# --- EmptySet ---
+
+
+"""
+    ⊆(∅::EmptySet{N}, X::LazySet{N}, [witness]::Bool=false)::Bool where {N<:Real}
+
+Check whether an empty set is contained in another set.
+
+### Input
+
+- `∅`       -- empty set
+- `X`       -- another set
+- `witness` -- (optional, default: `false`) compute a witness if activated
+               (ignored, just kept for interface reasons)
+
+### Output
+
+`true`.
+"""
+function ⊆(∅::EmptySet{N}, X::LazySet{N}, witness::Bool=false
+          )::Bool where {N<:Real}
+    if witness
+        return (true, N[])
+    end
+    return true
+end
+
+# disambiguity
+function ⊆(∅::EmptySet{N}, H::AbstractHyperrectangle{N}, witness::Bool=false
```

Comment:
sorry, why is that `EmptySet` is ambiguous with `AbstractHyperrectangle`?

Reply:
The method
`⊆(∅::EmptySet{N}, S::LazySet{N})`
is ambiguous with
`⊆(S::LazySet{N}, H::AbstractHyperrectangle{N})`.

Reply:
Got it, thanks. I wonder if Julia has some mechanism to ease this workaround, like allowing to invoke a given method. Maybe with [invoke](https://docs.julialang.org/en/v1/base/base/#Core.invoke).

Reply:
I know, but `invoke` is less efficient, which is why I try to avoid it.

Reply:
Less efficient in which sense? The call overhead should be negligible, no?

Reply:
Not if you call this function very often. In this case I agree, though.